docs/reference/commandline/system_prune.md

```diff
@@ -68,6 +106,12 @@ deleted: sha256:3a88a5c81eb5c283e72db2dbc6d65cbfd8e80b6c89bb6e714cfaaa0eed99c548
 Total reclaimed space: 13.5 MB
 ```
 
+> **Note**: The `--volumes` option was added in Docker 17.06.1. Older versions
+> of Docker include volumes by default. On older versions, run `docker container prune`,
```

Comment:
prune volumes by default, along with other Docker objects.

Reply:
Thanks, yes, that's better 👍